### PR TITLE
Add cluster-policy-controller container

### PR DESCRIPTION
**Pull request by deads2k**
https://github.com/openshift/cluster-kube-controller-manager-operator/pull/287 with working pull policy

### CODE REVIEW

File: bindata/v4.1.0/kube-controller-manager/pod.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ spec:
   containers:
   - name: cluster-policy-controller-REVISION
     image: quay.io/openshift/origin-cluster-policy-controller:v4.3
-    imagePullPolicy: {{ .ImagePullPolicy }}
+    imagePullPolicy: IfNotPresent
```
Comment (sallyom) [inline bindata/v4.1.0/kube-controller-manager/pod.yaml]: seriously? _that's_ what I was missing???

File: manifests/image-references [hunk 2]
```diff
@@ -10,3 +10,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-hyperkube:v4.0
+  - name: cluster-policy-controller
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-cluster-policy-controller:v4.3
```
Comment (soltysh) [inline manifests/image-references]: @deads2k why this is 4.3, where the one above is 4.0? Not to mention the fact it's pointing to docker.io/... ?

File: Makefile [hunk 3]
```diff
@@ -24,7 +24,8 @@ IMAGE_REGISTRY :=registry.svc.ci.openshift.org
 # $2 - image ref
 # $3 - Dockerfile path
 # $4 - context directory for image build
-$(call build-image,ocp-cluster-kube-controller-manager-operator,$(IMAGE_REGISTRY)/ocp/4.2:cluster-kube-controller-manager-operator, ./Dockerfile.rhel7,.)
+IMAGE_TAG?=$(IMAGE_REGISTRY)/ocp/4.2:cluster-kube-controller-manager-operator
+$(call build-image,ocp-cluster-kube-controller-manager-operator,$(IMAGE_TAG), ./Dockerfile.rhel7,.)
```
Comment (soltysh) [inline Makefile]: @sallyom please fix that in a followup.

Comment (sallyom) [reply Makefile]: yes, i saw that, mentioned in slack @deads2k can fix here, or whatever

File: manifests/image-references (hunk 2 above)
Comment (sallyom) [reply manifests/image-references]: right, was also wondering that, thanks for reminding me

**Comment by deads2k** [comment]
sally got hers updated and this is failing on the image not being substituted in payload now.

**Pull request closed without merging**